pkg/artifactory/resource_artifactory_user.go

```diff
@@ -10,9 +10,21 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
-	"github.com/jfrog/jfrog-client-go/artifactory/services"
```

Comment:
I'm kinda surprised that this client is still in the code base. @chb0github

Reply:
Then it's being used somewhere - you'd get a compile error if you had an unused import. @maheshjfrog delete it and see what breaks. It might require you C&P code into here. 

There maybe some lingering references to the jfrog client that ultimately should be removed